docs/intro_06_audits.md

```diff
@@ -4,11 +4,13 @@ title: Security audits
 sidebar_label: Security audits
 ---
 
-We take great care ensuring security of our smart contracts. Please find the audit reports of the Gnosis Safe smart contracts below:
+We take great care ensuring security of our smart contracts. Please find the audit reports of the Gnosis Safe smart contracts and the allowance module (spending limit) below:
 
 - [Gnosis Safe v0.0.1](https://github.com/gnosis/safe-contracts/blob/v1.1.1/docs/alexey_audit.md).
 - [Gnosis Safe v1.0.0](https://github.com/gnosis/safe-contracts/blob/v1.1.1/docs/Gnosis_Safe_Formal_Verification_Report_1_0_0.pdf).
 - [Gnosis Safe v1.1.0 & v1.1.1](https://github.com/gnosis/safe-contracts/blob/v1.1.1/docs/audit_1_1_1.md).
 - [Gnosis Safe v1.2.0](https://github.com/gnosis/safe-contracts/blob/v1.2.0/docs/audit_1_2_0.md).
 
+- [Allowance Module v0.1.0](https://github.com/gnosis/safe-modules/blob/master/allowances/AllowanceModuleAuditOct2020.md).
```

Comment:
I would use `https://github.com/gnosis/safe-modules/blob/47e2b486b0b31d97bab7648a3f76de9038c6e67b/allowances/AllowanceModuleAuditOct2020.md` to avoid that the link breaks when master is updated